projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -143,17 +204,32 @@ function asset_version( $version ) {
 		return apply_filters( 'jp_carousel_asset_version', $version );
 	}
 
-	function display_bail_message( $output = '' ) {
-		// Displays a message on top of gallery if carousel has bailed
+	/**
+	 * Displays a message on top of gallery if carousel has bailed.
+	 *
+	 * @param string $output Bail message output.
+	 *
+	 * @return string
+	 */
+	public function display_bail_message( $output = '' ) {
 		$message  = '<div class="jp-carousel-msg"><p>';
 		$message .= __( 'Jetpack\'s Carousel has been disabled, because another plugin or your theme is overriding the [gallery] shortcode.', 'jetpack' );
 		$message .= '</p></div>';
-		// put before gallery output
+		// put before gallery output.
 		$output = $message . $output;
 		return $output;
 	}
 
-	function check_if_shortcode_processed_and_enqueue_assets( $output ) {
+	/**
+	 * Allow third-party plugins or themes to force-enable Carousel.
+	 *
+	 * @since 1.9.0
+	 *
+	 * @param bool $output Should we force enable Carousel? Default to false.
+	 *
+	 * @return bool
```

Comment:
Ah I didn’t dig deep enough here, thanks for digging in and also making this suggestion.  I’d go with that, though one thing I'd wonder about is removing the info about maybe displaying a bail message just to make it slightly more concise (since it’s just a maybe). What would you think about that?